bin/spiral

```diff
@@ -9,8 +9,8 @@ declare(strict_types=1);
 
 foreach ([
              __DIR__ . '/../../../autoload.php',
-             __DIR__ . '/../vendor/autoload.php',
-             __DIR__ . '/vendor/autoload.php'
+             __DIR__ . '/../vendor_php/autoload.php',
```

Comment:
This is not correct. The vendor path will only change for local development, the actual project will still use vendor.

Reply:
done